js/private/js_run_binary.bzl

```diff
@@ -275,6 +298,31 @@ See https://github.com/aspect-build/rules_js/tree/main/docs#using-binaries-publi
             name = to_label(name),
         ))
 
+    # Configure run from execroot
+    if run_from_execroot:
+        fixed_env["JS_BINARY__RUN_FROM_EXECROOT"] = "1"
+
+        if allow_run_from_execroot_with_no_copy_data_to_bin:
```

Comment:
It's probably only applicable when `run_from_execroot` is set, but should we put this `if` outside the `run_from_execroot` `if` just so the `JS_BINARY__` vars directly align with the bzl params?

Reply:
Its only ever checked in the launcher if `JS_BINARY__RUN_FROM_EXECROOT` is set but it doesn't hurt to hoist it for alignment